patient-care/webapp/client/tools/limma.html

```diff
@@ -0,0 +1,145 @@
+<template name="listLimma">
+  <h1>
+    Limma
+
+    {{#semanticUIPopup selector=".help.circle.icon"}}
+      <i class="help circle icon"></i>
+      <p class="ui popup">
+        Create a genomic signature given two phenotypes with
+        Bioconductor's limma R package.
+      </p>
+    {{/semanticUIPopup}}
+  </h1>
+
+  <h2>Run a new analysis</h2>
+  {{#autoForm id="createLimma" schema=createLimmaSchema
+      type="method" meteormethod="createLimma"
+      class=(loadingIfFalse Template.subscriptionsReady)}}
+    {{> afQuickField name="value_type" type="select"
+        placeholder="Select a value type..."}}
+
+    {{#if sampleGroupOptions}}
+      {{> afQuickField name="experimental_sample_group_id"
+          options=sampleGroupOptions search=true type="select"
+          placeholder="Select an experimental sample group..."}}
+      {{> afQuickField name="reference_sample_group_id"
+          options=sampleGroupOptions search=true type="select"
+          placeholder="Select a reference sample group..."}}
+    {{/if}}
+
+    {{> afQuickField name="top_genes_count" value=1000
+        placeholder="Top N differential genes to be used in the signature"}}
+
+    {{#if duplicateSamples}}
```

Comment:
Good catch! Actually this was because I never implemented the thing I meant to implement. I copied it over from the paired analysis code and meant to change it to not allow job creation if there were any samples that were in both sample groups, which causes Limma to crash. I'm going to change it to a `TODO` that returns false for now.